_clients/data-prepper/migrate-open-distro.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Migrating from Open Distro Data Prepper
+parent: Data Prepper
+nav_order: 2
+---
+
+# Migrating from Open Distro Data Prepper
+
+Beginning with Data Prepper 1.1, there is only one distribution of Data Prepper - Open Search Data Prepper. This document helps existing users migrate from the old Open Distro Data Prepper to OpenSearch Data Prepper.
+
+## Change your Pipeline Configuration
+
+The `elasticsearch` sink has changed to `opensearch`. You will need to change your existing pipeline to use the `opensearch` plugin instead of `elasticsearch`.
+
+Please note that, while the plugin is titled `opensearch`, it remains compatible with Open Distro and ElasticSearch 7.x.
+
+## Update Docker Image
+
+The Open Distro Data Prepper Docker image was located at `amazon/opendistro-for-elasticsearch-data-prepper`. You will need to change this value to `opensearchproject/data-prepper`.
+
+## More Information
```

Comment:
Is this a topic?

Reply:
Removed this file from this PR because I am working with this file in a different branch.

https://github.com/opensearch-project/documentation-website/pull/1485